README.md

```diff
@@ -168,6 +168,10 @@ This means that, in order to start up, Edge mode needs to know where the upstrea
 
 By default, Edge mode uses an in-memory cache to store the features it fetches from the upstream node. However, you may want to use a more persistent storage solution. For this purpose, Edge supports either Redis or a backup file, which you can configure by passing in either the `--redis-url` or `--backup_folder` command line argument, respectively. On start-up, Edge checks whether the persistent backup option is specified, in which case it uses it to populate its internal caches. This can be useful when your Unleash server is unreachable.
 
+Persistent storage is useful for high availability and resilience. If Edge using persistent storage is restarted, it will not have to synchronize with upstream before it is ready. Persistent storage is only read on start-up and valid until Edge is able to synchronize with upstream. This makes it so there is no need for purging cache. After Edge is synchronized with upstream it will periodically save its in-memory state to the persistent storage.
```

Comment:
```suggestion
Persistent storage is useful for high availability and resilience. If an Edge instance using persistent storage is restarted, it will not have to synchronize with upstream before it is ready. The persistent storage is only read on startup and will only be used until Edge is able to synchronize with upstream. Because of this, there is no need to purge the cache. After Edge has synchronized with an upstream, it will periodically save its in-memory state to the persistent storage.
```